src/Utilities/PaymentUtils.res

```diff
@@ -295,3 +295,13 @@ let updateDynamicFields = (arr: Js.Array2.t<PaymentMethodsRecord.paymentMethodsF
   }
   newArr
 }
+
+let getPaymentMethodName = (~paymentMethodType, ~paymentMethodName) => {
+  if paymentMethodType == "bank_debit" {
+    paymentMethodName->Js.String2.replace("_debit", "")
+  } else if paymentMethodType == "bank_transfer" {
+    paymentMethodName->Js.String2.replace("_transfer", "")
+  } else {
+    paymentMethodName
+  }
+}
```

Comment:
can you check how we were doing this for the existing bank_transfer methods ?
we are not reverting this back for those right .. please make use of the same here too

Reply:
For the existing bank transfers, it is handled in different files, ACHBankDebit and so on, PaymentMethodWrapper is not used

Reply:
okay since you are already on this.. can you make those files also utilise this function

Reply:
bumping this up